src/UI/RenderConfig.elm

```diff
@@ -44,3 +45,9 @@ isMobile (RenderConfig { deviceClass }) =
 isPortrait : RenderConfig -> Bool
 isPortrait (RenderConfig { deviceOrientation }) =
     deviceOrientation == Element.Portrait
+
+
+rootAttributes : RenderConfig -> List (Attribute msg)
```

Comment:
As `elm-ui` defines the root elemen as `layout` maybe we should call this `layoutAttributes`?

Reply:
To be honest, idk if here is the best place to put it, but at the same time, I don't know any place else to put it

Reply:
> maybe we should call this `layoutAttributes`?

I originally did, then I thought "root" was more explicit.
Now considering that it's made to be used with Element's layout and it alone, it feels obvious that it should be called `elLayoutAttributes`.

~I wrote something very very wrong in that comment~